recipes/logfault/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class PackageConan(ConanFile):
+    name = "logfault"
+    description = "Simple to use, header only C++ library for application-logging on all major platforms."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jgaa/logfault"
+    topics = ("logging", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+           "Visual Studio": "15",
+           "msvc": "191",
+           "gcc": "8",
+           "clang": "7",
+           "apple-clang": "12",
+        }
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables['LOGFAULT_BUILD_TESTS'] = False
+        tc.generate()
```

Comment:
We're not building the library, so this is probably not needed. Did you do this to be able to call cmake on your own?

Reply:
I used it only for testing, and without tests, it's not needed.
I have pushed an update without the generate method.